STANDARDS.md

```diff
@@ -29,6 +29,19 @@ Make sure your plugin follows the SoC design principle and has different functio
 
 Make effective use of namespacing since it will allow co-existence of ambiguous names / classes with the same name and help avoid collisions.
 
+### Recommended Code Style
+Format Java files with the Eclipse JDT formatter, using the [Spotless Gradle](https://github.com/diffplug/spotless/tree/master/plugin-gradle) plugin.
```

Comment:
What is JDT?

Reply:
Java Development Tools. Will push the change